lib/src/Dscp/CLI/Common.hs

```diff
@@ -110,9 +124,9 @@ netServParamsParser = NetServParams <$> peersParser <*> ourZTNodeIdParser
 ---------------------------------------------------------------------------
 
 parseNetAddr :: String -> Either String NetworkAddress
-parseNetAddr str =
-    let host = toText $ takeWhile (/= ':') str
-        portS = drop (length host + 1) str
+parseNetAddr st =
```

Comment:
Such change.

Reply:
It conflicted with `str` from `opt-parse` :man_shrugging: